plugins/catalog-import/api-report.md

```diff
@@ -226,7 +237,58 @@ export const StepInitAnalyzeUrl: ({
   onAnalysis,
   analysisUrl,
   disablePullRequest,
-}: Props_3) => JSX.Element;
+  exampleEntityFilepath,
+}: Props_4) => JSX.Element;
+
+// Warning: (ae-missing-release-tag) "StepperProvider" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export type StepperProvider = {
```

Comment:
I am pretty sure that we shouldn't export this component. It's highly internal and not stable enough to make a lot of sense for others to consume. And as you see, it pretty heavily pollutes the API surface - at the bottom you see warnings that even _more_ types and stuff would need to be exported as well to make this complete.

I say revert this one.

Reply:
I ended up exporting this because I needed the type definitions for custom stepper for `ImportStepper` component. I understand that the definitions might not be stable enough but that makes me wonder if `generateStepper` prop should be there after all? 

I guess I could do something like the following to provide `exampleEntityFilepath` and to rely on implicit types which wouldn't require exposing types used in `generateStepper`:
```tsx
<ImportStepper
  generateStepper={(flow, defaults) => {
    const stepper = defaultGenerateStepper(flow, defaults);

    return {
      ...stepper,
      analyze: (state, { apis }) => ({
        stepLabel: <StepLabel>Select URL</StepLabel>,
        content: (
          <StepInitAnalyzeUrl
            key="analyze"
            analysisUrl={state.analysisUrl}
            onAnalysis={state.onAnalysis}
            disablePullRequest={
              !apis.catalogImportApi.preparePullRequest
            }
            exampleEntityFilepath="https://github.com/acme-corp/our-awesome-api/blob/main/anvil.yaml"
          />
        ),
      }),
    };
  }}
/>
```
I understand this is very fragile but haven't come up with a better way to provide this example entity filepath without needing to re-write the whole stepper logic.

Reply:
Reverted the change, `exampleEntityFilepath` can still be passed down to `StepInitAnalyzeUrl` as in the example above